engines/agi/text.cpp

```diff
@@ -563,11 +583,17 @@ void TextMgr::statusDraw() {
 		clearLine(_statusRow, 15);
 
 		charAttrib_Set(0, 15);
-		charPos_Set(_statusRow, 1);
+		if (!_vm->isLanguageRTL())
+			charPos_Set(_statusRow, 1);
+		else
+			charPos_Set(_statusRow, 21);
 		statusTextPtr = stringPrintf(_systemUI->getStatusTextScore());
 		displayText(statusTextPtr);
 
-		charPos_Set(_statusRow, 30);
+		if (!_vm->isLanguageRTL())
+			charPos_Set(_statusRow, 30);
+		else
+			charPos_Set(_statusRow, 1);
```

Comment:
ditto

Reply:
Note that the magic `30` is in the original code.